scripts/us_bea/states_gdp/import_data.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pulls data from the US Bureau of Economic Analysis (BEA) on quarterly GDP
+per State. Saves output as CSV.
+
+    Typical usage:
+
+    python3 import_data.py
+"""
+import json
+from absl import app
+from absl import flags
+import requests
+import pandas as pd
+
+# TODO (fpernice): switch to POST python call.
+MY_KEY = 'D431C2CE-8BD2-4D9E-AD7A-00F95CAB60CE'
+QUARTERLY_GDP_TABLE = 'SQGDP1'
```

Comment:
Will leave QUARTERLY_GDP_TABLE as global and put everything else inside the class, changing __init__ to take in the table name instead of the raw request. That way one can init an instance by calling `loader = StateGDPDataLoader(table=QUARTERLY_GDP_TABLE)`